USAGE.md

```diff
@@ -0,0 +1,2 @@
+- [SendGrid Docs](https://sendgrid.com/docs/API_Reference/SMTP_API/index.html)
```

Comment:
This is a great start. Can you please add some more information? The [SMTPAPI-PHP](https://github.com/sendgrid/smtpapi-php/blob/master/USAGE.md) usage file is more in-line with what we were looking for, can you give that a look and make a similar file here?